www/src/js/controllers/store.js

```diff
@@ -1,21 +1,87 @@
+var $ = require('jquery');
 var Backbone = require('backbone');
+Backbone.$ = $;
 var Marionette = require('backbone.marionette');
 var Radio = require('backbone.radio');
 var StoreLayoutView = require('../views/store.js');
-var Snaplist = require('../collections/snaplist.js');
+var Sections = require('../collections/sections.js');
+var Bask = require('../collections/snaplist.js');
```

Comment:
for consistency across views, but I reverted that, ... wasn't such a significant change